contracts/src/ISequencerInbox.sol

```diff
@@ -31,6 +31,9 @@ interface ISequencerInbox {
     /// @dev Thrown when sequencer tries to append an empty batch
     error EmptyBatch();
 
+    /// @dev Thrown when the txn data length does not match
```

Comment:
nit: this is too specific / goes into impl details, consider
```
    /// @dev Thrown when overflow occurs reading txBatch (likely due to malformed txLengths)
    error TxBatchOverflow();
```

Reply:
Would you like to change the error name to `TxBatchOverflow` or keep `TxDataOverflow` or something else like `TxBatchDataOverflow` ?

@ujvl